scripts/autostake.mjs

```diff
@@ -315,7 +315,7 @@ class Autostake {
   }
 
   getNetworksData() {
-    const networksData = fs.readFileSync('src/networks.json');
+    const networksData = fs.readFileSync('src/networks.local.json');
```

Comment:
You shouldn't need to change this. The local file is just for your local autostaking bot currently, and it's used automatically if it exists.